src/Resources/config/services.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    contao_community_alliance.event_dispatcher.populator:
```

Comment:
Maybe use cca as shortcut here. As I said would be nice to discuss our naming conventions.Question  ;-)